deploy/helm/distributed-compute-operator/templates/hooks.yaml

```diff
@@ -69,7 +69,7 @@ metadata:
     "helm.sh/hook": pre-install,pre-upgrade
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
 spec:
-  backoffLimit: 0
+  backoffLimit: 3
```

Comment:
Any reason to set this to 3 instead of removing the limitation and using K8s' default value? I think it's 6.

Reply:
Guess not, nope. Updated.